NEWS.md

```diff
@@ -8,6 +8,10 @@
   parsing. Settings parsed later will now override earlier values and
   the `!` exclusion logic for lists was removed. Assigning the empty
   string to a list can be used to clear previously assigned values.
+- Configuration files now accept a `DefaultSetting=value` that will be
+  used if the `Setting=` is not set elsewhere. This allows for such things
+  as `DefaultRelease=stable` in the main configuration and `Release=unstable`
+  to be used in a preset.
```

Comment:
This is a bad example, now that the first assignment doesn't take precedence anymore, you can just do `Release=unstable` in the preset file and it will override the main configuration. Where this is useful is to override mkosi's default value so that a value can be specified explicitly in the CLI but if it's ,not set in the CLI the correct default value is used.

Reply:
Yeah that sounds good